packages/core/src/connectors/coinbaseWallet.ts

```diff
@@ -68,7 +67,12 @@ export class CoinbaseWalletConnector extends Connector<
         ),
       }
     } catch (error) {
-      if (/user closed modal/i.test((<ProviderRpcError>error).message))
+      console.log(error)
```

Comment:
Just double checking - is this console log meant to be here?

Reply:
nope! good catch